src/Day1.hs

```diff
@@ -113,6 +118,15 @@ clearTileMap c = fromList
 colorFromRoomIndex :: RoomIndex -> Color
 colorFromRoomIndex (RoomIndex idx) = colors !! (idx `mod` len)
   where
-    colors = colorWheel2
+    colors = rainbow
```

Comment:
use `colorWheel2`

Reply:
Okay will do. Going to fix now real quick